src/extension/app.tsx

```diff
@@ -68,14 +84,65 @@ chrome.runtime.sendMessage({ fetch: true }, function (response) {
             }
         }
             
-        store.dispatch({
+        Store.dispatch({
             type: Types.SelectSession,
             payload: session
         });
 
-        store.dispatch({
+        Store.dispatch({
             type: Types.SelectImpression,
             payload: session[activeIndex]
         });
     }
-});
\ No newline at end of file
+});
+
+function compareVersions(v1: string, v2: string): number | null {
```

Comment:
maybe just use this ([https://www.npmjs.com/package/compare-versions](https://www.npmjs.com/package/compare-versions)) instead? Looks like it is super small and presumably handles some cases we aren't handling yet

Reply:
Agreed. Surprising that i didn't come across this npm package myself (and i was looking) - would have saved me 20mins or so :)